### PR TITLE
Add new QRDA Report to Export

### PR DESCRIPTION
**Pull request by sjpadgett**
Fixes #

#### Short description of what this resolves:
Patient with all Measures

### CODE REVIEW

**Comment by sjpadgett** [comment]
Memory issues with export. Doesn't appear to be from mustache.

image

**Comment by sjpadgett** [comment]
somebody fix the tests. don't think it's me

**Comment by stephenwaite** [comment]
Brady's going to look into tonight

**Comment by bradymiller** [comment]
looking into the testing at https://github.com/openemr/openemr/issues/5054

File: src/Cqm/QrdaControllers/QrdaReportController.php [hunk 1]
```diff
@@ -69,41 +66,71 @@ public function getCategoryIReport($pid, $measures, $type = 'xml')
...
+            $zip->setArchive($save_path);
+            $zip->compress($zip_directory);
+        } elseif ($bypid) {
             foreach ($pids as $pid) {
                 $meta = sqlQuery("Select `fname`, `lname` From `patient_data` Where `pid` = ?", [$pid]);
-                $file = $measure_directory . "/{$meta['fname']}_{$meta['lname']}." . $type;
-                $content = $this->getCategoryIReport($pid, $measure, $type);
-                $directory = $GLOBALS['OE_SITE_DIR'] . DIRECTORY_SEPARATOR . 'documents' . DIRECTORY_SEPARATOR . 'temp';
-                file_put_contents($directory . DIRECTORY_SEPARATOR . $measure . "_{$meta['lname']}_{$meta['fname']}." . 'xml', $content);
+                $file = $zip_directory . "/{$meta['fname']}_{$meta['lname']}." . $type;
```
Comment (bradymiller) [inline src/Cqm/QrdaControllers/QrdaReportController.php:121]: potential issue here where fname or lname could be set to traverse directories (in cases like this, would run those variables through the convert_safe_file_dir_name() function which is always available in the library/sanitize.inc.php script)

File: interface/modules/zend_modules/module/Carecoordination/src/Carecoordination/Controller/EncounterccdadispatchController.php [hunk 2]
```diff
@@ -103,8 +103,12 @@ public function indexAction()
             $view = $this->params('view');
             $pids = explode('|', $combination);
             $measures = $_REQUEST['report_measures'] ?? "";
-            if (is_array($measures) && empty($measures[0])) {
-                $measures = ''; // defaults to all current.
+            if (is_array($measures)) {
+                if (empty($measures[0])) {
+                    $measures = ''; // defaults to all current one per patient.
+                } elseif (!empty($measures[0] == 'all')) {
```
Comment (sjpadgett) [inline interface/modules/zend_modules/module/Carecoordination/src/Carecoordination/Controller/EncounterccdadispatchController.php:109]: I sometimes wonder what the heck I was thinking the next day!
!empty($measures[0] == 'all')

**Pull request merged**